Posts/2021/02/Get-Services, PowerShell 7 VS Windows PowerShell 5.1

```diff
@@ -0,0 +1,150 @@
+---
+post_title: Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+username: farisnt@gmail.com
+Catagories: PowerShell, Windows Services
+Summary: Show how to get services information using both Windows PowerShell 5.1 and PowerShell 7
+---
+
+# Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+
```

Comment:
The H1 is not needed for the blog. The post_title is automatically displayed.

Reply:
Got it